src/main/scala/ru/org/codingteam/keter/scenes/inventory/InventoryScene.scala

```diff
@@ -35,6 +36,36 @@ class InventoryScene(parentScene: Scene, inventory: Inventory)
   private def backpackCategories = viewModel.backpackCategories
   private def currentFolderItems = viewModel.currentFolderItems
   private def currentItemInfo = viewModel.currentItemInfo
+
+  private def folderView(shape: Rectangle) = {
+    val model = currentFolderItems
+    def defaultAction() = {
+      model.selectedItem foreach { case item =>
```

Comment:
My idea here was to make default actions vary in future, but currently it have no use.